protocol/currency_store.go

```diff
@@ -0,0 +1,852 @@
+package protocol
+
+type Currency struct {
+	Id          string `json:"id"`
+	ShortName   string `json:"shortName"`
+	Name        string `json:"name"`
+	Symbol      string `json:"symbol"`
+	Emoji       string `json:"emoji"`
+	ImageSource string `json:"imageSource"` // used by desktop app
+	IsToken     bool   `json:"isToken"`
+	IsPopular   bool   `json:"isPopular"` // to help the UI to show popular currencies in separate section
+}
+
+// This list serves as a common currencies data which can be used across platforms
+// This data is moved from status-desktop
+// https://github.com/status-im/status-desktop/blob/2b415bdc5359f424fbfcabf1b5319bc3a8595921/ui/imports/shared/stores/CurrenciesStore.qml#L30
+var currencyList = []*Currency{
+	&Currency{
+		Id:          "usd",
+		ShortName:   "USD",
+		Name:        "United States Dollar",
+		Symbol:      "$",
+		Emoji:       "🇺🇸",
+		ImageSource: "../../assets/twemoji/svg/1f1fa-1f1f8.svg",
```

Comment:
Does this relative path fit for both mobile and desktop apps? 

Isn't it better to have `unicode` param from twemoji, like this:

```suggestion
		Unicode: "1f1fa-1f1f8",
```
That allows client to use it in the way it wants and keep assets in the location which is the best for it.

Reply:
> Does this relative path fit for both mobile and desktop apps?

Nope. The mobile doesn't use Twemoji. This relative path is used by the Desktop. So, I copied it over to not break it. I can change it to `unicode`, but for the cryptocurrencies, it will be empty. Is that fine?

The desktop has a different relative path for cryptocurrency image asset `../../../../imports/assets/png/tokens/ETH.png` (compared to Twemoji path `../../assets/twemoji/svg/1f1ec-1f1e7.svg`)

In Mobile, we skipped the displaying flag next to the name for this milestone 2.30 (later we could use the emoji). We only display the image for cryptocurrency (which depends on `id`).

Reply:
> The desktop has a different relative path for cryptocurrency image asset ../../../../imports/assets/png/tokens/ETH.png (compared to Twemoji path ../../assets/twemoji/svg/1f1ec-1f1e7.svg)

Please, remove `ImageSource`, then add `unicode` property and set it only for twemoji related icons, when we're working on the related desktop app issue for this, we will check if `unicode` is empty and then load appropriate icon from appropriate location based on the `ShortName`. So don't worry, just make it to be generic. ;)

Also please if there is no already an issue for this in the desktop repo, could you please create one and add the link to this PR as a reference?

Reply:
Yeah we don't directly refer to local graphics resources from status-go. If we reeeally want to make those resources part of status-go, we could provide them using the `MediaServer`. But everywhere else we just make resources part of the client and have some ID to connect an entry that comes from status-go to some file that's client-side.

Reply:
@saledjenic - I have updated the data with your suggestion. Removed `ImageSource` and added `Unicode` for Twemoji.